h/static/styles/annotation-share-dialog.scss

```diff
@@ -0,0 +1,130 @@
+.share-dialog-wrapper.open .annotation-share-dialog {
+  display: flex;
+  flex-direction: column;
+}
+
+.annotation-share-dialog {
+  display: none;
+  position: absolute;
+  right: 0;
+  bottom: 130%;
+  z-index: 1;
+  background: $white;
+  border: 1px solid $gray-lighter;
+  border-radius: 2px;
+  width: 200px;
+  font-size: $body1-font-size;
+  cursor: default;
+  box-shadow: 0px 0px 4px 0px rgba(0,0,0,0.15);
+
+  &:after, &:before {
+    top: 100%;
+    right: 10px;
+    border: solid transparent;
+    content: " ";
+    height: 0;
+    width: 0;
+    position: absolute;
+    pointer-events: none;
+  }
+
+  &:after {
+    border-color: rgba(255, 255, 255, 0);
+    border-top-color: $white;
+    border-width: 5px;
+    margin-right: -5px;
+  }
+
+  &:before {
+    border-color: rgba(211, 211, 211, 0);
+    border-top-color: $gray-lighter;
+    border-width: 6px;
+    margin-right: -6px;
+  }
+}
+
+.annotation-share-dialog-msg {
+  color: $gray-light;
+  margin: -5px 10px 10px 10px;
+  line-height: 15px;
+  font-size: 11px;
```

Comment:
The font size and line height should be defined in the typography palette in `variables.scss` if possible